ansibleflow/app.py

```diff
@@ -50,13 +50,29 @@ def setup_argument_parser():
     )
 
     run_parser = subparsers.add_parser('run')
-    run_parser.add_argument('--env', type=str, default='default')
     run_parser.add_argument(
         'run_action',
-        nargs='*',
+        type=str,
         default=True,
         help=argparse.SUPPRESS
     )
+    run_parser.add_argument(
+            '--env',
+            type=str,
+            default='default')
+    run_parser.add_argument(
+            "-v",
+            "--verbosity",
+            action="count",
+            default=0,
+            help="increase output verbosity")
```

Comment:
I couldn't figure out how to get the remaining text not parsed by ansible-flow to be just available to pass onto ansible-playbook. I would prefer that but argparse seems to want to parse - and -- options anyway even if you use the '*' or 'REMAINING' options for nargs. :\